src/client/components/app/Navbar/index.tsx

```diff
@@ -15,21 +15,21 @@ const StyledOptionList = styled(OptionList)`
 const StyledNavbarActions = styled.div`
   display: grid;
   grid-auto-flow: column;
-  grid-gap: 0.8rem;
+  grid-gap: 1.6rem;
```

Comment:
we should check all buttons and mobile in another pr later, just to have it in mind